torch/export/graph_signature.py

```diff
@@ -283,11 +283,15 @@ def user_inputs(self) -> Collection[Union[int, float, bool, None, str]]:
         return tuple(user_inputs)
 
     # Graph node names of pytree-flattened outputs of original program
+    # For joint-graph purposes, will include the loss output.
     @property
     def user_outputs(self) -> Collection[Union[int, float, bool, None, str]]:
         user_outputs: List[Union[int, float, bool, None, str]] = []
         for s in self.output_specs:
-            if s.kind != OutputKind.USER_OUTPUT:
+            if s.kind not in [
+                OutputKind.USER_OUTPUT,
+                OutputKind.LOSS_OUTPUT,
```

Comment:
Why we need changing this?

Reply:
For verifier purposes this in a way includes the loss output: https://github.com/pytorch/pytorch/blob/d276688da6c1881d058670624fda4589f68b4571/torch/_export/verifier.py#L427

We don't have anything specific for training in the verifier yet.. ok maybe this is just a lazy change.